README.md

```diff
@@ -102,23 +102,27 @@ Support of TLS on the Server is managed through a pkcs12 der file. The documenta
 
 ## DNS-over-TLS and DNS-over-HTTPS
 
-DoT and DoH are supported. This is accomplished through the use of one of `native-tls`, `openssl`, or `rustls` (only `rustls` is currently supported for DoH). The Resolver requires valid DoT or DoH resolvers being registered in order to be used.
+The Resolver requires valid DoT or DoH resolvers being registered in order to be used.
```

Comment:
I'm not sure how or why this might be accurate -- it seems wrong to me? @divergentdave any thoughts?

Reply:
I think the original meaning might have been "you can use DoH/DoT, but only if you configure a DoH/DoT upstream name server. Deleting the first sentence thus changes the meaning of this paragraph.
```suggestion
The Resolver supports DoT and DoH. These protocols can be selected and configured when forwarding to another server.
```